tests/basics/op_precedence.py

```diff
@@ -37,7 +37,12 @@
 # BAD: (-2)**2 = 4
 print(-2**2)
 # OK: 2**(-1) = 0.5
-print(2**-1)
+try:
+    eval('0.5')  # is float supported?
+except SyntaxError:
+    print('0.5')  # skip test
+else:
+    print(2**-1)
```

Comment:
Good catch!  What it's really testing is order or parsing of `** -`, not FP, so I think it's best to change it to something that works without FP.  How about `2**-0`?  Or even `2**-(-1)`.

Reply:
Ah, of course! `2**-0` doesn't use floating point so that works.